include/openthread/border_routing.h

```diff
@@ -498,6 +498,32 @@ void otBorderRoutingDhcp6PdSetEnabled(otInstance *aInstance, bool aEnabled);
  */
 otBorderRoutingDhcp6PdState otBorderRoutingDhcp6PdGetState(otInstance *aInstance);
 
+/**
+ * Function pointer callback which is invoked when state of request/release a prefix via DHCPv6 Prefix Delegation (PD)
+ * for Thread interface changed.
+ *
+ * @param[in] aRequest  Whether to request RA.
+ * @param[in] aContext  A pointer to arbitrary context information.
+ *
+ */
+typedef void (*otBorderRoutingRequestDhcp6PdCallback)(bool aRequest, void *aContext);
```

Comment:
It's not immediately obvious the purpose of "requesting RA" just reading this API doc alone. Can you provide more documentation to make it obvious why the callback is useful to the developer?

Reply:
updated... not sure if I need to explain more about why we need to handle the state change...